__tests__/user/user.query.test.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-client');
+
+require('dotenv').config();
+
+let token, userId
+
+describe('queries', () => {
+    beforeAll(async () => {
+        const register = await client.mutate({
+            mutation: gql`
+                mutation {       
+                    registerUser(user: {
+                        firstName: "Test" 
+                        lastName: "User"
+                        email: "test.email@gmail.com"
+                        password: "12345678Te"
+                    }) {
+                        _id
+                        firstName
+                        lastName
+                        email
+                        role
+                        registrationDate
+                        credentials {
+                            tokenPass
+                        } 
+                    }
+                }
+            `
+        })
+
+        userId = register.data.registerUser._id
+
+        const authRes  = await client.mutate({
+            mutation: gql`
+                mutation {       
+                    loginUser(user: {
+                        email: "test.email@gmail.com"
+                        password: "12345678Te"
+                    }) {
+                        token
+                    }
+                }
+            `
+        })
+        token = authRes.data.loginUser.token
+    });
+
+    afterAll(async () => {
+        await client.mutate({
+            mutation: gql`
+                mutation($userId: ID!) {       
+                    deleteUser(id: $userId) {
+                        _id
+                    }
+                }
+            `,
+            variables: {
+                userId
+            }
+        })
```

Comment:
Please, add a check that operation has been successful by checking that `_id` is returned back in body

Reply:
fixed